src/components/TempTodo/TempTodo.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  tempTodo: Todo | null;
+};
+export const TempTodo: React.FC<Props> = ({ tempTodo }) => {
+  return (
+    <div data-cy="Todo" className="todo">
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={false}
+        />
+      </label>
+
+      <span data-cy="TodoTitle" className="todo__title">
+        {tempTodo?.title}
+      </span>
+
+      {/* Remove button appears only on hover */}
+      <button type="button" className="todo__remove" data-cy="TodoDelete">
+        ×
+      </button>
+
+      {/* overlay will cover the todo while it is being deleted or updated */}
+      <div data-cy="TodoLoader" className="modal overlay is-active">
+        <div className="modal-background has-background-white-ter" />
+        <div className="loader" />
+      </div>
```

Comment:
you can just use the TodoItem for tempTodo instead of creating a separate component